addons/mail/static/src/messaging.js

```diff
@@ -142,6 +142,9 @@ export class Messaging {
             chatPartnerId: false,
             isAdmin: false,
             canLeave: data.canLeave || false,
+            isRenameable() {
```

Comment:
I think type does not change (is that right?), so you can just set it like directly:
```js
isRenameable: (['chat', 'channel', 'group'].includes(type),
```